quickstart/oneclick_instruction_box.md

```diff
@@ -1,15 +1,17 @@
 ### Prerequisites
 
-1. Set up Java version 8 or higher on your machine. For instructions, consult a Java environment setup tutorial, like [this](https://www.tutorialspoint.com/java/java_environment_setup.htm).
-1. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+- [Go](https://go.dev/) version 1.19 or higher ```
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- An [Azure Data Explorer Cluster](https://learn.microsoft.com/en-us/azure/data-explorer/).
+- An Azure Data Explorer Database. You can create a Database in your Azure Data Explorer Cluster using the [Azure Portal](https://learn.microsoft.com/en-us/azure/data-explorer/create-cluster-database-portal).
 
 ### Instructions
 
-1. Download the **DOWNLOAD_LINK** as a ZIP file.
-1. Extract the app source code.
+1. Download the app files from this GitHub repo.
+1. Modify the `kusto_sample_config.json` file, changing `KustoUri`, `IngestUri` and `DatabaseName` appropriately for your cluster.
```

Comment:
This is inveresed.
These instructions should appear on README.md and not on the one click instructions where the file is auto generated

Reply:
They appear as well on the README. or did you mean something else?